x-pack/metricbeat/module/aws/rds/rds.go

```diff
@@ -215,6 +215,52 @@ func (m *MetricSet) getDBInstancesPerRegion(svc rdsiface.ClientAPI) ([]string, m
 					Value: *tag.Value,
 				})
 		}
+
+		// WIP - Get tags from cluster
+		describeDBClustersInput := &rds.DescribeDBClustersInput{
+			DBClusterIdentifier: dbInstance.DBClusterIdentifier,
+		}
+
+		reqCluster := svc.DescribeDBClustersRequest(describeDBClustersInput)
+		outputCluster, err := reqCluster.Send(context.TODO())
+		if err != nil {
+			return nil, nil, errors.Wrap(err, "Error DescribeDBClustersRequest")
+		}
+
+		for _, dbCluster := range outputCluster.DBClusters {
+			// Get tags for each RDS instance
+			listTagsInput := rds.ListTagsForResourceInput{
```

Comment:
Can we combine these two for loops somehow? https://github.com/elastic/beats/pull/19524/files#diff-e02bb39466ac2176434b8314321236ddR188 It would be good to not have the duplicate code here 😄